src/main/java/utils/filereaders/CsvFileReader.java

```diff
@@ -9,9 +10,13 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class CsvFileReader implements CustomFileReader {
 
+    private static final Logger logger = LogManager.getLogger(CsvFileReader.class);
```

Comment:
UPPER_CASE the variable name, all static constant variables must have uppercase letters